src/main/java/org/ecocean/media/MediaAsset.java

```diff
@@ -429,7 +429,17 @@ public void setLabels(ArrayList<String> l) {
     }
     public void addLabel(String s) {
         if (labels == null) labels = new ArrayList<String>();
-        if (!labels.contains(s)) labels.add(s);
```

Comment:
was this not getting persisted when you just plain `add()`ed it?  i think i have seen this behavior.  i guess if the duplicate thing is working lets not rock the boat! ha